### PR TITLE
Task/APPS-3091 — Enrolled View

### PR DESCRIPTION
**Pull request by giuliacampana**
Screen Shot 2021-11-17 at 4 09 28 PM

### CODE REVIEW

File: chronicle/Views/EnrolledView.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            VStack(alignment: .leading) {
+                AppHeader()
+                Text("Study ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.studyId) ?? "").padding(.bottom)
+                Text("Participant ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.participantId) ?? "").padding(.bottom)
+                if (defaults.string(forKey: UserSettingsKeys.organizationId) != nil) {
+                    Text("Organization ID:").padding(.bottom)
+                    Text(defaults.string(forKey: UserSettingsKeys.organizationId) ?? "").padding(.bottom)
+                }
```
Comment (UnsungHero97) [inline chronicle/Views/EnrolledView.swift]: I remember approving the other pull request where the org id is made optional. I think we should revert that and make the org id required.

File: chronicle/Views/EnrollmentForm.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                AppHeader()
+                Text("Do you have an organization ID?")
+                    .padding(.bottom)
+                RadioButtonGroup(hasOrgId: $enrollmentViewModel.withOrgId)
+
+                if enrollmentViewModel.withOrgId == true {
+                    InputFieldView(label: "Enter your Organization ID", inputId: $enrollmentViewModel.organizationId, invalidInput: $enrollmentViewModel.invalidOrganizationId)
+                }
+                InputFieldView(label: "Enter your Study ID", inputId: $enrollmentViewModel.studyId, invalidInput: $enrollmentViewModel.invalidStudyId)
+                InputFieldView(label: "Enter your Participant ID", inputId: $enrollmentViewModel.participantId, invalidInput: $enrollmentViewModel.invalidParticipantId)
```
Comment (UnsungHero97) [inline chronicle/Views/EnrollmentForm.swift]: Please use the same order across views. Either ...

1. participant id
2. study id
3. org id

... or ...

1. org id
2. study id
3. participant id

I prefer the first.

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: chronicle/Views/EnrolledView.swift (hunk 1 above)
Comment (giuliacampana) [reply chronicle/Views/EnrolledView.swift]: okay, gotcha—so just to clarify, we are making the org id required on the enrollment form, as well as displaying it by default here on the enrolled screen?

File: chronicle/Views/EnrollmentForm.swift (hunk 2 above)
Comment (giuliacampana) [reply chronicle/Views/EnrollmentForm.swift]: oh ya, good point. thanks!

File: chronicle/ContentView.swift [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        EnrollmentForm()
+    }
+}
+
+
+struct ContentView_Previews: PreviewProvider {
+    static var previews: some View {
+        ContentView()
+        ContentView().preferredColorScheme(.dark)
+    }
```
Comment (anzioka) [inline chronicle/ContentView.swift]: I think this can safely be deleted.

File: chronicle/Views/EnrollmentForm.swift [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            .padding(.horizontal)
+            .textFieldStyle(.roundedBorder)
+        }
+    }
+}
+
+struct EnrollmentForm_Previews: PreviewProvider {
+    static var previews: some View {
+        EnrollmentForm()
+    }
```
Comment (anzioka) [inline chronicle/Views/EnrollmentForm.swift]: There is already a EnrollmentView, right? This seems to be a duplicate

**Review by anzioka — CHANGES_REQUESTED** [review]

**Review by UnsungHero97 — DISMISSED** [review]

File: chronicle/Views/EnrolledView.swift [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading) {
+                AppHeader()
+                Text("Participant ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.participantId) ?? "").padding(.bottom)
+                Text("Study ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.studyId) ?? "").padding(.bottom)
+                Text("Organization ID:").padding(.bottom)
+                Text(defaults.string(forKey: UserSettingsKeys.organizationId) ?? "").padding(.bottom)
```
Comment (anzioka) [inline chronicle/Views/EnrolledView.swift]: No need to use UserDefaults here. You can access the organizationId, participantId and studyId directly from the viewModel object

**Review by anzioka — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**